WalletWasabi.Gui/Program.cs

```diff
@@ -112,13 +112,40 @@ private static async void AppMainAsync(string[] args)
 				if (!(ex is OperationCanceledException))
 				{
 					Logger.LogCritical(ex);
+					Global.CrashReporter.SetException(ex);
 				}
 
-				await Global.DisposeAsync().ConfigureAwait(false);
+				await DisposeAsync();
```

Comment:
might break OSX... if this is like a Main method?

Reply:
It should not.

Reply:
Our current conventions regarding that:
- In the Main method, we cannot use async.
- In AppMainAsync we should not change Threads.

Both are satisfied.